### PR TITLE
Update ClientRuntime to latest version

### PR DESCRIPTION
**Pull request by cormacpayne**
## Description

Fix for https://github.com/Azure/azure-powershell/issues/5380

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#making-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/ServiceFabric/Commands.ServiceFabric.Test/Commands.ServiceFabric.Test.csproj [hunk 1]
```diff
@@ -127,11 +127,7 @@
     <Reference Include="System.Net.Http.Primitives">
       <HintPath>..\..\..\packages\Microsoft.Net.Http.2.2.28\lib\net45\System.Net.Http.Primitives.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Rest.ClientRuntime, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Rest.ClientRuntime.2.3.8\lib\net452\Microsoft.Rest.ClientRuntime.dll</HintPath>
     </Reference>
```
Comment (maddieclayton) [inline src/ResourceManager/ServiceFabric/Commands.ServiceFabric.Test/Commands.ServiceFabric.Test.csproj]: Need to remove these two end tags.

**Comment by cormacpayne** [comment]
Need to include change log updates once the release branch has been merged back into `preview`.

**Comment by cormacpayne** [comment]
On-demand: https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/656/
Sign job: https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/294/

**Comment by cormacpayne** [comment]
On-demand: https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/658/
Sign job: https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/298/

**Comment by cormacpayne** [comment]
On-demand: https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/670/ (and 671)
Sign job: https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/316/

**Comment by cormacpayne** [comment]
I'm not sure why the Travis build is stuck, but the latest result from this PR passed: https://travis-ci.org/Azure/azure-powershell/builds/358480461

**Pull request merged**